zkstack_cli/crates/zkstack/src/commands/chain/args/create.rs

```diff
@@ -106,8 +108,14 @@ impl ChainCreateArgs {
                     .ask()
             });
 
+        let l1_network = l1_network.unwrap_or_else(|| {
+            self.l1_network.unwrap_or_else(|| {
+                PromptSelect::new(MSG_L1_NETWORK_PROMPT, L1Network::iter()).ask()
+            })
+        });
```

Comment:
Looks a bit odd. Why do we have two places for l1 network?

Reply:
`l1_network` as a parameter of the function is the one coming from ecosystem config. If exists we use that one. `l1_network` as attribute (`self.l1_network`) comes from the command arguments: provided by the user. If there is no ecosystem and the user didn't provide `l1_network` then there is a prompt